internal/validator/beef/beef_validator_test.go

```diff
@@ -211,3 +211,100 @@ func getPolicy(satoshisPerKB uint64) *bitcoin.Settings {
 	policy.MinMiningTxFee = float64(satoshisPerKB) / 1e8
 	return policy
 }
+
+func Test_cumulativeCheckFees(t *testing.T) {
```

Comment:
I think the size of this test could be reduced if it were table based like this

```
func Test_cumulativeCheckFees(t *testing.T) {
	tt := []struct {
		name      string
		minedBeef string
	}{
		{
			name:      "mined parents only - valid fee",
			minedBeef: "0100beef.....",
		},
		{
			...
		},
	}

	for _, tc := range tt {
		t.Run(tc.name, func(t *testing.T) {
			hex, _ := hex.DecodeString(tc.minedBeef)
			beefTx, _, _ := beef.DecodeBEEF(hex)

			feeQuote := bt.NewFeeQuote()
			setFees(feeQuote, 1)

			// then
			vErr := cumulativeCheckFees(beefTx, feeQuote)

			// assert
			require.Nil(t, vErr)
		})
	}
```

Reply:
You right, it even looks better now.

Reply:
Changed to table based